### PR TITLE
Update Contributing.md

### PR DESCRIPTION
**Pull request by lafentres**
Updating version 2.7.8 to 2.7.11 to match version in pyenv install instruction.

I noticed this when I was setting up my development environment and thought I'd go ahead and update it.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=424) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=424) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-python?pullRequest=424) it.</sub>

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ pyenv install 2.7.11
 pyenv install 3.4.3
 pyenv install 3.5.0
 python setup.py install
-pyenv local 3.5.0 3.4.3 2.7.8 2.6.9
+pyenv local 3.5.0 3.4.3 2.7.11 2.6.9
```
Comment (mbernier) [inline CONTRIBUTING.md]: Would it make sense to change this to `2.7.*` ?

Comment (lafentres) [reply CONTRIBUTING.md]: That makes sense to me. There are actually some other issues with this file as well. I had a hard time getting set up and ended up just using Docker. I'd be happy to look a little closer at the areas I had issues with and propose some changes, if you think that would be useful.

Comment (mbernier) [reply CONTRIBUTING.md]: Yes, that would be absolutely amazing! Please do! 
Also, if you had any issues with Docker, please suggest changes there too. 

Our theory is that if you had issues, someone else is going to - so any time you see something, please let us know! We would much rather have 1000 PRs fixing small silly issues than 1 customer having trouble using the library. 

THANK YOU!!

**Comment by lafentres** [comment]
I've added some things I think will make things a little clearer to first-timers getting set up. But I have a few questions:

1. I never got this command to work: 
```source venv/bin/activate ```
There's no direct mention of venv anywhere and I just get the following error:
```-bash: venv/bin/activate: No such file or directory```. I'm not sure how to resolve this one

2. I would like to make it a little more clear which steps you still need to run with the Docker image and which you only have to run if you are installing and running locally. 
My initial thoughts are that, with the Docker image, you still need to do the "Environment Variables" section but nothing else (so none of the install/setting up under the "Testing" section). Is that correct?

**Comment by thinkingserious** [comment]
Thanks for the great feedback @lafentres!

My apologies for the slow response. We are currently working on each PR from oldest to newest as we try to catch up from an amazing Hacktoberfest where we received over 1,000 PRs!

Thank you for your continued support and patience.

With Best Regards,

Elmer

**Comment by thinkingserious** [comment]
Hello @lafentres,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**